components/startadojo/Hero.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="c-hero">
+    <h1 class="c-hero__header1">{{ $t('Let\'s start a Dojo to help engage young people near you') }}</h1>
+    <div class="c-hero__step-cards">
+      <div class="c-hero__step-card">
+        <img class="c-hero__image" src="~/static/images/doc.svg" />
+        <h1 class="c-hero__step-header c-hero__orange">{{ $t('Step 1') }}</h1>
+        <p class="c-hero__step-text"><span class="c-hero__step-text-first-word c-hero__orange">{{ $t('Register ') }}</span>{{ $t('with us as a Champion so we can support you as you build your club.') }}</p>
```

Comment:
That doesn't work for translations. 
I would recommend using js to do that instead (translate the whole sentence then highlight on created the specific keyword)

Reply:
dont